git-bootstrap.yaml

```diff
@@ -39,11 +39,10 @@ pipeline:
           expected-sha256: 9a93b2b7dfdac77ceba5a558a580e74667dd6fede4585b91eefb60f03b72df23
       - runs: |
           ./configure \
-          --prefix=/usr \
-          --libdir=/lib
+          --prefix=/usr
       - uses: autoconf/make
       - runs: |
-          make install pkgconfigdir="/usr/lib/pkgconfig"
+          make install
```

Comment:
What's the downstream impact of this change? Just curious

Reply:
of the pkgconfigdir specifically? or the whole change? by default pkconfigdir is configured to be ${libdir}, but that's setup to be ${prefix} so it's going to be the paths that are defined during build time arguments. 

see [zlib/Makefile.in](https://github.com/madler/zlib/blob/51b7f2abdade71cd9bb0e7a373ef2610ec6f9daf/Makefile.in#L51C1-L51C13)